packages/react-storage/src/components/StorageBrowser/controls/ActionsListControl.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { ControlProps } from './types';
+import { useResolvedComposable } from './hooks/useResolvedComposable';
+import { ActionsList } from '../composables/ActionsList';
+import { useActionsList } from './hooks/useActionsList';
+import { ViewElement } from '../context/elements';
+
+export const ActionsListControl = ({
+  className,
+}: ControlProps): React.JSX.Element => {
+  const props = useActionsList();
+  const ResolvedActionsList = useResolvedComposable(ActionsList, 'ActionsList');
+
+  return (
+    <ViewElement className={className}>
```

Comment:
We don't need the positional divs anymore right?

Reply:
Artifact of copy/paste. I'll remove from every control in a followup PR.